py/dml/objects.py

```diff
@@ -283,29 +283,20 @@ def set_traits(self, traits):
         self.traits = traits
 
 class Device(CompositeObject):
-    __slots__ = ('staticvars', 'initdata', 'static_idx')
+    __slots__ = ('initdata')
```

Comment:
I prefer tuple form, `('initdata',)`. IIRC python has a hack to support string form, if you insist on this then please remove brackets.

Reply:
Oh, oops, yeah, you're quite right, I simply forgot.